src/coreComponents/unitTests/fluidFlowTests/testFluidStatistics.cpp

```diff
@@ -0,0 +1,1197 @@
+/*
```

Comment:
could you please rename it to test**Flow**Statistics.cpp?

Reply:
It is indeed a better name